doc/LICENSES.md

```diff
@@ -32,6 +32,7 @@ Bellow follows a list of all code licenses used in Cardinal and linked submodule
 | Extratone               | GPL-3.0-or-later      | |
 | Fehler Fabrik           | GPL-3.0-or-later      | |
 | Fundamental             | GPL-3.0-or-later      | |
+| Geodesics               | GPL-3.0-or-later      | |
```

Comment:
https://github.com/MarcBoule/Geodesics/blob/master/plugin.json#L5 disagrees with this line, and this is a blocker

Reply:
Ah yes, its LICENSE does say GPL3 as well. Why is the "or later" part a blocker?

Reply:
Just read up on it, thanks for clarifying. OK, well we'll at least have the PR until they do change license :)

Reply:
Current proper term to use is `GPL-3.0-only` btw, this is a SPDX specification.